cli/commands/preview/create.ts

```diff
@@ -60,14 +63,21 @@ async function runCommand( siteFolder: string, outputFormat?: OutputFormat ): Pr
 	}
 }
 
-export const registerCommand: RegisterCommand = ( program ) => {
-	program
-		.command( 'go [folder]' )
-		.description(
-			__( 'Create a preview site from the specified folder (defaults to current directory)' )
-		)
-		.action( async ( siteFolder: string = process.cwd() ) => {
-			const options = program.opts();
-			await runCommand( siteFolder, options.outputFormat );
-		} );
+export const registerCommand = ( yargs: StudioArgv ) => {
+	return yargs.command( {
+		command: 'go [folder]',
```

Comment:
```suggestion
		command: 'go [folder=cwd]',
```
While we're at it, I think this would add clarity for users.

Reply:
For some reason this breaks the command. (it always uses the cwd value)